tests/Unit/Controllers/AdminControllerTest.php

```
@@ -2109,6 +2109,74 @@ public function testSaveSettingsPersistsHlQualityEnabled(): void
+                if (str_contains($sql, 'SELECT * FROM organisations')) {
+                    return $this->stmt($org);
+                }
+                if (str_contains($sql, 'UPDATE organisations SET')) {
+                    $captured = $params;
+                    return $updateStmt;
+                }
+                return $this->stmt(false);
+            }
+        );
+
+        $r = $this->makePostRequest([
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $r. Configured minimum length is 3. (undefined)

(ShortVariable)